doc/source/tutorial-quickstart-pytorch.rst

```diff
@@ -10,7 +10,12 @@ Quickstart PyTorch
 ..  youtube:: jOmmuzMIQ4c
    :width: 100%
 
-In this tutorial we will learn how to train a Convolutional Neural Network on CIFAR10 using Flower and PyTorch.
+.. admonition:: Disclaimer
+    :class: important
+
+    The Quickstart PyTorch video uses slightly different Flower commands than this tutorial. Please follow the :doc:`Upgrade to Flower Next <how-to-upgrade-to-flower-next>` guide to convert commands shown in the video.
```

Comment:
```suggestion
    The Quickstart PyTorch video uses slightly different Flower commands than this tutorial. Please follow the :doc:`Upgrade Flower <how-to-upgrade-to-flower-next>` guide to convert commands shown in the video.
```

Reply:
We should avoid advertising "Flower Next" to prevent confusion among users who may be searching for "Next Flower." Instead, we should refer to versions of Flower prior to 1.9.

Reply:
💯 I'll update the series of quickstart docs next week.